contracts/test/PoolFDTLosses.t.sol

```diff
@@ -0,0 +1,314 @@
+
+        // Fund the loan by both pool delegates.
+        pat.fundLoan(address(pool),  address(custom_loan), address(dlFactory), usdc.balanceOf(pool.liquidityLocker()));
+        pam.fundLoan(address(pool2), address(custom_loan), address(dlFactory), usdc.balanceOf(pool2.liquidityLocker()));
+
+        // Drawdown of the loan
+        drawdown(custom_loan, bob, totalDepositAmt);
+
+        /// Time warp to make a payment.
+        hevm.warp(custom_loan.nextPaymentDue());
+
+        // Make partial payment of loan
+        doPartialLoanPayment(custom_loan, bob);
+
+        {
+            // Claim the funds.
+            uint256[7] memory claimInfo1 = pat.claim(address(pool),  address(custom_loan), address(dlFactory));
+            uint256[7] memory claimInfo2 = pam.claim(address(pool2), address(custom_loan), address(dlFactory));
+
+            // withdraw interest by the lps.
+            leo.withdrawFunds(address(pool));
+            liz.withdrawFunds(address(pool));
+            lex.withdrawFunds(address(pool2));
+            lee.withdrawFunds(address(pool2));
+
+            // Verify the interest.
+            withinDiff(
+                getClaimedInterestNetOfFees(claimInfo1[1], pool.delegateFee(), pool.stakingFee()),
+                usdc.balanceOf(address(leo)) + usdc.balanceOf(address(liz)),
+                1
+            );
+            
+            withinDiff(
+                getClaimedInterestNetOfFees(claimInfo2[1], pool2.delegateFee(), pool2.stakingFee()),
+                usdc.balanceOf(address(lex)) + usdc.balanceOf(address(lee)),
+                1
+            );
+        
+
+            // Time warp to default
+            hevm.warp(custom_loan.nextPaymentDue() + globals.defaultGracePeriod() + 1);
+
+            // Setting slippage high enough to let it through the liquidations.
+            gov.setMaxSwapSlippage(7000);
+
+            // At least one of the pool delegates should be able to trigger a default
+            assertTrue(
+                pat.try_triggerDefault(address(pool), address(custom_loan), address(dlFactory)) ||
+                pam.try_triggerDefault(address(pool2), address(custom_loan), address(dlFactory)),
+                "Should be able to trigger Loan default"
+            );
+            
+            // Check for successful default.
+            assertTrue(uint8(custom_loan.loanState()) == 4, "Unexpected Loan state");
+
+            claimInfo1 = pat.claim(address(pool), address(custom_loan), address(dlFactory));
+            claimInfo2 = pam.claim(address(pool2), address(custom_loan), address(dlFactory));
+
+            assertTrue(claimInfo1[6] > 0,        "Loan doesn't has default suffered");
+            assertTrue(pool.poolLosses() > 0, "Pool 1 losses should be greater than 0");
+            withinPrecision(pool.recognizableLossesOf(address(leo)) + pool.recognizableLossesOf(address(liz)),  pool.poolLosses(), 5);
```

Comment:
Lets try `withinDiff` here, should be rounding errors

Reply:
I always prefer withinDiff, with fuzzing tests the precision decimals can vary widely, but the diff is usually the same